packages/charting/src/components/VerticalBarChart/VerticalBarChartPage.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { ComponentPage, ExampleCard, IComponentDemoPageProps, PropertiesTableSet } from '@uifabric/example-app-base';
+
+import { VerticalBarChartBasicExample } from './examples/VerticalBarChart.Basic.Example';
+
+const VerticalBarChartBasicExampleCode = require('!raw-loader!@uifabric/charting/src/components/VerticalBarChart/examples/VerticalBarChart.Basic.Example.tsx') as string;
```

Comment:
may conflict with @kenotron pr.

Reply:
This happens to be okay because it is not OUFR package itself. So it will need to be refactored before being promoted to OUFR.